lib/ffmpeg/encoderbase.py

```diff
@@ -33,6 +33,7 @@ class Encoder(PropertyHandler, BaseFormatMapper):
   encoded       = property(lambda s: s._encoded)
   osencoded     = property(lambda s: filepath2os(s.encoded))
   encoded_ext   = property(lambda s: s.props["encoded_ext"])
+  skip_checks   = property(lambda s: s.props["skip_checks"])
```

Comment:
This should not be a property of the `Encoder` class.  In fact, you are not even using this property from the `EncoderTest` class.  If you were, then in that class it would be `self.encoder.skip_checks`, not `self.skip_checks`.   Nonetheless, it does not belong here.

Please remove this property from the Encoder classes.